packages/twenty-server/src/modules/company/repositories/company.repository.ts

```diff
@@ -67,11 +69,11 @@ export class CompanyRepository {
     );
 
     await this.workspaceDataSourceService.executeRawQuery(
-      `INSERT INTO ${dataSourceSchema}.company (id, "domainName", name, "addressAddressCity", position)
+      `INSERT INTO ${dataSourceSchema}.company (id, "${companyDomainNameColumnName}", name, "addressAddressCity", position)
       VALUES ($1, $2, $3, $4, $5)`,
       [
         companyToCreate.id,
-        companyToCreate.domainName,
+        'https://' + companyToCreate.domainName,
```

Comment:
Why do we add 'https://' here but we don't in `getExistingCompaniesByDomainNames`? The get and the create should have a consistant api, either `companyToCreate` and `domainNames` should both contain `https://` or they shouldn't.

Reply:
I had not paid attention to that, thanks. 
I am replacing the query to compare the domain names without the  'https://' or 'http://' prefix; this seems safer than adding the prefix the domainNames and comparing. (although when we migrate the existing data we are adding https:// so all existing data should have a https:// prefix but still)

Reply:
yet to test though, waiting on your help on how to trigger sync again

Reply:
tested !